09-documenting.Rmd

```diff
@@ -4,7 +4,58 @@ R has a substantial body of documentation, comprising help files for the core pa
+This chapter is about the ways people can contribute to R's documentation, with guidance on how to do that. Any time you feel that you can clarify or fill gaps in existing documentation, your contribution is welcome and appreciated. If you find it difficult to deal with the markup formatting language used in the source files, you can ask for help with that part too. Please do not let the material in this chapter stand between the documentation and your desire to help out. However, not every good faith effort to change or extend the documentation will be accepted - sometimes the suggested changes may be incorrect; other times, while a change in wording may make some things clearer and easier to understand, the finer details of some corner case may become less clear, leading to the suggested changes being declined or modified by a member of R Core before applying them (if they agree the issue is important enough to fix).
+
+## Helping with documentation
+
+Maintaining the accuracy of R's documentation and keeping a high level of quality takes a lot of effort. Community members, like you, help with writing, editing, and updating content, and these contributions are appreciated and welcomed.
+
+Looking at pre-existing documentation source files can be very helpful when getting started.
+
+You can directly search for [documentation issues/bugs on Bugzilla](https://bugs.r-project.org/buglist.cgi?component=Documentation&list_id=22501&product=R&resolution=---). Issues vary from typos to unclear documentation and items lacking documentation.
+
+If you see a documentation issue that you would like to tackle, you can leave a comment on the issue saying you are going to try to solve the issue and mention roughly how long you think you will take to do so (this allows others to take on the issue if you happen to forget or lose interest).
+
+## Reporting documentation bugs
+ <!--
+discussion in slack (random channel)
+-->
+
+For improvements of the R-manuals, reporting typos or bugs in the R-manuals, or issues related to packages maintained by the R core team, read the [reporting bugs guide](#ReportBug).
+
+For reporting bugs or typos on webpages or documentation about a particular package where it is not maintained by the R core team, you can find the maintainer of a package by using the `maintainer("package-name")` function.
```

Comment:
If the package has an issue tracker or specific way to report bugs that should be used rather than contacting the maintainer directly - this is all document in the reporting bugs section so we can refer to that.
```suggestion
To report bugs or typos in the documentation of a package that is not maintained by the R Core Team, follow the guidance for [reporting bugs in contributed packages](#nonRCorePkgBug).
```